django/librarian/models.py

```diff
@@ -88,15 +88,13 @@ class Library(models.Model):
     # Last access time manually updated when library is queried through Library Q&A
     accessed_at = models.DateTimeField(auto_now_add=True)
 
-    chat = models.OneToOneField(
-        "chat.Chat", on_delete=models.CASCADE, null=True, blank=True
-    )
     order = models.IntegerField(default=0)
     is_public = models.BooleanField(default=False)
     is_default_library = models.BooleanField(default=False)
```

Comment:
The distinction between public, default, and personal is becoming less clear. Maybe "type" or "audience" and the options are public/private. And if private, then we rely on the row level access controls. The owner of the object always has permissions. (Again, something we can adjust another time.)

Reply:
Thanks, would be nice to brainstorm this. To better explain, "is_default" means the global default for Otto. i.e. all users have this selected initially when they go to Q&A mode. (Corporate library). "is_public" indicates that the library is viewable to all Otto users. There are different permissions and warnings for editing when a library faces all users. (data steward role, etc)